app/src/features/Batches/BatchList.tsx

```diff
@@ -130,6 +82,34 @@ const BatchList = (): JSX.Element => {
     }
   );
 
+  const convertDateToTime = (batchStart: string, batchEnd: string) => {
+    const timeBatchStart = new Date(batchStart).getTime();
+    const timeBatchEnd = new Date(batchEnd).getTime();
+    const duration = timeBatchEnd - timeBatchStart;
+    const hours = Math.floor((duration / (1000 * 60 * 60)) % 24);
+    const minutes = Math.floor((duration / (1000 * 60)) % 60);
+    return { hours, minutes };
```

Comment:
you shouldn't have top do this by hand. see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date

Reply:
![giphy](https://user-images.githubusercontent.com/47388675/133286580-780613bd-d434-42d6-a8a0-7dc1e4226e5b.gif)